src/tools/kdb/basename.h

```diff
@@ -0,0 +1,33 @@
+/**
+ * @file
+ *
+ * @brief Header for basename command
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ */
+
+#ifndef ELEKTRA_KDB_BASENAME_H
+#define ELEKTRA_KDB_BASENAME_H
+
+#include <kdb.h>
+
+/**
```

Comment:
This is very minor, but what kind of function documentation do we follow in general? For some I see that we are doing documentation in the `*.c` files and some things are documented in `*.h`.